src/Controllers/DatabaseController.js

```diff
@@ -360,6 +360,15 @@ const relationSchema = {
   fields: { relatedId: { type: 'String' }, owningId: { type: 'String' } },
 };
 
+const maybeTransformUsernameAndEmailToLowerCase = (object, className, options) => {
+  if (className === '_User' && options.forceEmailAndUsernameToLowerCase) {
+    const toLowerCaseFields = ['email', 'username'];
+    toLowerCaseFields.forEach(key => {
+      if (typeof object[key] === 'string') object[key] = object[key].toLowerCase();
```

Comment:
```suggestion
      if (typeof object[key] === 'string') {
        object[key] = object[key].toLowerCase();
      }
```

Reply:
Is there a lint rule for this? This pattern should really be prevented.